docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| sink.properties.row_delimiter     | No           | \n                | The row delimiter for CSV-formatted data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| sink.properties.column_separator  | No           | \t                | The column separator for CSV-formatted data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| sink.properties.max_filter_ratio  | No           | 0                 | The maximum error tolerance of the stream load. It's the maximum percentage of data records that can be filtered out due to inadequate data quality. Valid values: 0 to 1. Default value: 0. See [Stream Load](https://docs.starrocks.io/en-us/latest/sql-reference/sql-statements/data-manipulation/STREAM%20LOAD) for details.                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/7e8cee1c-309d-46d4-a76e-36f3f16ccb50)
Search for all "stream load" and see if they are correctly written in the doc.